### PR TITLE
Release

### PR DESCRIPTION
**Pull request by nelsonic**
This can be shipped.

### CODE REVIEW

**Comment by nelsonic** [comment]
@iteles please review and let me know if this is unclear BEFORE MERGING. Thanks!

File: README.md [hunk 1]
```diff
@@ -31,22 +41,23 @@ npm install ignored --save
...
 
-There are actually ***4 ways*** to use this module in your project:
+There are actually ***3 ways*** to use this module in your code:
 
 #### 1. Sync (*Without* passing a .gitignore file as parameter)
 
 The **simplest way** to use this module is to let it figure out where your
-project's .gitignore file is and return the list ***syncronously*** at the
+project's .gitignore  
+file is and return the list ***syncronously*** at the
```
Comment (iteles) [inline README.md]: Typo here at 'sync**h**ronously'

Comment (nelsonic) [inline README.md]: Please correct online. thanks.

**Comment by iteles** [comment]
Changed a couple of typos and merged :+1: :blush:

**Comment by nelsonic** [comment]
Did you read my comment?

**Comment by iteles** [comment]
@nelsonic To let you know if there were any questions before merging? Yes - I mentioned just the typos and fixed them online as requested before merging. The module itself is super clear, especially with the tests to follow for implementation.

**Comment by nelsonic** [comment]
Cool. just cause I don't know if your merge included my latest commits...

**Pull request merged**